modules/system/rolls/cast-test.js

```diff
@@ -190,7 +192,7 @@ export default class CastTest extends TestWFRP {
 
     // TODO handle all tooltips (when they are added) in one place
     // TODO Fix weird formatting in tooltips (indenting)
-    this.result.tooltips.miscast = this.result.tooltips.miscast.join("\n")
+    this.result.tooltips.miscast = "<ul style='text-align: left'>" + this.result.tooltips.miscast.map(t => `<li>${t}</li>`).join("") + "</ul>";
```

Comment:
This may not be needed, I have not investigated it but I believe there is a `data-tooltip-class` property we can use to style foundry tooltips

Reply:
i guess we can remove the style, but i assume that to make it look pretty, <ul> structure has to be preserved?